gateway-ha/pom.xml

```diff
@@ -24,7 +24,7 @@
         <dep.jeasy.version>4.1.0</dep.jeasy.version>
         <dep.mockito.version>5.13.0</dep.mockito.version>
         <dep.okhttp3.version>4.12.0</dep.okhttp3.version>
-        <dep.mysqlconnector.version>8.3.0</dep.mysqlconnector.version>
+        <dep.mysqlconnector.version>9.0.0</dep.mysqlconnector.version>
```

Comment:
Trino user reported 8.4.0 caused performance regression. https://github.com/trinodb/trino/issues/21945
I'm not sure whether this version is safe to use.

Reply:
Hmm.. good question.. I guess we would need to try with 9.0.0 in Trino as well and decide from there.

Reply:
https://dev.mysql.com/doc/relnotes/mysql/9.0/en/news-9-0-0.html
> This release is no longer available for download. It was removed due to a critical issue that could stop the server from restarting following the creation of a very large number of tables (8001, or more). Please upgrade to MySQL 9.0.1 instead.

I don't think we should use this version.